python_modules/dagster/dagster_tests/definitions_tests/test_external_assets.py

```diff
@@ -295,6 +295,6 @@ def test_external_assets_with_dependencies() -> None:
     defs = Definitions(assets=external_assets_from_specs([upstream_asset, downstream_asset]))
     assert defs
 
-    assert defs.get_implicit_global_asset_job_def().asset_layer.asset_deps[
-        AssetKey("downstream_asset")
-    ] == {AssetKey("upstream_asset")}
+    assert defs.get_asset_graph().asset_dep_graph["upstream"][AssetKey("downstream_asset")] == {
+        AssetKey("upstream_asset")
+    }
```

Comment:
Cleaner to test dependencies in the definitions asset graph than the global asset job, which is an implementation detail.